### PR TITLE
Remove trailing whitespace from minutes 2020-10-10

### PR DESCRIPTION
**Pull request by yongping827**

### CODE REVIEW

File: docs/minutes/20201010.md [hunk 1]
```diff
@@ -30,9 +30,10 @@ Jingjing:
 - Make `Transaction` abstract
 - Generic `FindCommand` that works based on the current `UiState`
 
-Siddarth: 
+Siddarth:
 - Adding left side panel in the main window
 - Change menu tabs to tab panel
-- Center categories 
+- Center categories
+
```
Comment (ianyong) [inline docs/minutes/20201010.md:37]: Is this line break intended?

**Review by ianyong — CHANGES_REQUESTED** [review]

**Review by ianyong — APPROVED** [review]
LGTM!

**Review by zhaojj2209 — APPROVED** [review]
LGTM!

**Review by wltan — APPROVED** [review]
LGTM

**Pull request merged**